microsetta_interface/implementation.py

```diff
@@ -2486,6 +2510,17 @@ def get_update_sample(*, account_id=None, source_id=None, sample_id=None):
         sample_output['date'] = ""
         sample_output['time'] = ""
 
+    if sample_output['sample_site'] == "Cheek":
+        # Format date and time to be JavaScript-friendly
+        if sample_output['barcode_meta'][
+            'sample_site_last_washed_date'
```

Comment:
I'm unsure whether these keys are guaranteed to exist as the private API can return an empty dict? If they are not assured to exist then they could they be accessed through `get`?

https://github.com/biocore/microsetta-private-api/blob/d9653bc1059957f8afab23c4ca804810c365255b/microsetta_private_api/repo/sample_repo.py#L423

Reply:
All three keys _should_ exist for any cheek sample, and their absence would indicate an unexpected state. However, using `get` doesn't hurt anything and mitigates the minimal potential for an error.